LICENSE

```diff
@@ -7,7 +7,7 @@ Licensed Work:        Omni
                       The Licensed Work is (c) 2024 Sidero Labs, Inc.
 Additional Use Grant: None
 
-Change Date:          2028-08-16
```

Comment:
we probably shouldn't change this on the release branch (not critical, but still)

release branch is a snapshot of `main`, so we should rather update `main`